app/src/main/java/org/oppia/android/app/settings/profile/ProfileRenameActivityPresenter.kt

```diff
@@ -104,9 +103,7 @@ class ProfileRenameActivityPresenter @Inject constructor(
 
   private fun handleAddProfileResult(result: AsyncResult<Any?>, profileId: Int) {
     if (result.isSuccess()) {
-      val intent = ProfileEditActivity.createProfileEditActivity(activity, profileId)
```

Comment:
The hint is that we are no longer creating new Intent and launching the activity, instead we are using the previous intent.
So our test cases are written for checking for creating of new Intent, therefore learn how to test intents and then look in the tests of these activities.

Reply:
`System.logW: A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.java.lang.Throwable: Explicit termination method 'dispose' not called`
I tried to run the tests and the following error is thrown. The tests have no error but something else does. 
can you have a look at this?
@yash10019coder

Reply:
please add Screenshots of the error and the full error text